packages/gatsby-source-graphql/README.md

```diff
@@ -150,6 +150,49 @@ module.exports = {
 }
 ```
 
+## Composing Apollo Links for production network setup
+
+The plugin provides `createLink` option that you can use to enhance your network stack:
+add error handling, retries, timeouts, etc. Check out [this article](https://medium.com/@joanvila/productionizing-apollo-links-4cdc11d278eb)
+for an excellent explanation of how it works.
+
```

Comment:
Could you add something about what the parameters are for the `createLink` option?

Reply:
I've added a comment in the example. We also have some examples of this option at the beginning of the readme.